nano/templates/shuttle_mainpulator.tmpl.txt

```diff
@@ -0,0 +1,133 @@
+<ui-tabs tabs='{{data.tabs}}'>
+	<tab name='Status'>
+		<status/>
+	</tab>
+	<tab name='Templates'>
+		<templates/>
+	</tab>
+	<tab name='Modification'>
+		{{#if data.selected}}
+		<modification/>
+		{{/if}}
+		{{#if !data.selected}}
+		<span class='bad'>No shuttle selected.</span>
+		{{/if}}
+	</tab>
+</ui-tabs>
+
+<ui-display title='Selected: {{data.selected.name}}'>
+  {{#if data.selected.description}}
+    <ui-section label='Description'>{{data.selected.description}}</ui-section>
+  {{/if}}
+  {{#if data.selected.admin_notes}}
+    <ui-section label='Admin Notes'>{{data.selected.admin_notes}}</ui-section>
+  {{/if}}
+</ui-display>
+
+{{#if data.existing_shuttle}}
+  <ui-display title='Existing Shuttle: {{data.existing_shuttle.name}}'>
+    Status: {{data.existing_shuttle.status}}
+    {{#if data.existing_shuttle.timer}}
+      ({{data.existing_shuttle.timeleft}})
+    {{/if}}
+    <ui-button action='jump_to'
+      params='{"type": "mobile", "id": "{{data.existing_shuttle.id}}"}'>
+      Jump To
+    </ui-button>
+  </ui-display>
+{{/if}}
+
+{{#if !data.existing_shuttle}}
+  <ui-display title='Existing Shuttle: None'></ui-display>
+{{/if}}
+
+<ui-button action='preview'
+  params='{"shuttle_id": "{{data.selected.shuttle_id}}"}'>
+  Preview
+</ui-button>
+<ui-button action='load'
+  params='{"shuttle_id": "{{data.selected.shuttle_id}}"}'
+  style='danger'>
+  Load
+</ui-button>
+
+<ui-display title='Status'>
+</ui-display>
+
+<ui-display title='Selected: {{data.selected.name}}'>
+  {{#if data.selected.description}}
+    <ui-section label='Description'>{{data.selected.description}}</ui-section>
+  {{/if}}
+  {{#if data.selected.admin_notes}}
+    <ui-section label='Admin Notes'>{{data.selected.admin_notes}}</ui-section>
+  {{/if}}
+</ui-display>
+
+{{#if data.existing_shuttle}}
+  <ui-display title='Existing Shuttle: {{data.existing_shuttle.name}}'>
+    Status: {{data.existing_shuttle.status}}
+    {{#if data.existing_shuttle.timer}}
+      ({{data.existing_shuttle.timeleft}})
+    {{/if}}
+    <ui-button action='jump_to'
+      params='{"type": "mobile", "id": "{{data.existing_shuttle.id}}"}'>
```

Comment:
`"id"` should be `"shuttle_id"`, if `"shuttle_id"` is required by all `Topic` calls of the manipulator